content/en/docs/security/otel-collector-configuration-best-practices.md

```diff
@@ -0,0 +1,139 @@
+---
+title: OpenTelemetry Collector Configuration Best Practices
+linkTitle: Collector config
+description:
+  Follow best practices to securely configure OpenTelemetry Collector.
+weight: 20
+---
+
+When setting up configuration for OpenTelemetry (OTel) Collector, consider the
+following practices to better secure your OTel Collector instance.
+
+<!--
+TODO: SHOULD only enable the minimum required components. What are those?
+--->
+
+## Receivers and Exporters
+
+- Receivers and Exporters can be either push or pull-based. In either case, the
+  connection established SHOULD be over a secure and authenticated channel.
+- Unused receivers and exporters SHOULD be disabled to minimize the attack
+  vector of the Collector.
```

Comment:
```suggestion
- Receivers and exporters can be either push or pull-based. In either case, the
  connection established SHOULD be over a secure and authenticated channel.
- Unused receivers and exporters SHOULD be disabled to minimize the attack
  vector of the Collector.
```

I know specs capitalize verbs in RFC style, but I wouldn't do in docs. Could you lowercase all instances?

Reply:
Also, please write `exporters` with lower case 'e'.

Reply:
Agreed, as long as we do not make a definition that "OpenTelemetry Collector Exporters" are a fixed term for us, let's have them lowercase

Reply:
Fixed. Thanks all!